espgazer/main.cpp

```diff
@@ -0,0 +1,214 @@
+/*
+ * SPDX-FileCopyrightText: 2010-2022 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: CC0-1.0
+ */
+
+#include <gfx/gfx.h>
+#include <gfx/gfx_helpers.h>
+#include <inttypes.h>
+
+#include <cstdio>
+#include <cstring>
+#include <string>
+
+#include "esp_chip_info.h"
+#include "esp_flash.h"
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+#include "sdkconfig.h"
+
+using namespace antioch::gfx;
+#define CHECK_ERROR(expr)         \
+  if (expr != Result::eSuccess) { \
+    printf("%s failed\n", #expr); \
+  }
+
+extern "C" void app_main(void) {
+  printf("Hello world!\n");
+
+  /* Print chip information */
+  esp_chip_info_t chip_info;
+  uint32_t flash_size;
+  esp_chip_info(&chip_info);
+  printf("This is %s chip with %d CPU core(s), %s%s%s%s, ", CONFIG_IDF_TARGET, chip_info.cores,
```

Comment:
does this go to a different place than ESP_LOG?

Reply:
like alog in android. use that instead of printf bc that can get compiled away in prod

Reply:
ok